frontend/src/metabase/components/ItemsTable/BaseItemsTable/BaseItemsTable.unit.spec.tsx

```diff
@@ -110,6 +110,7 @@ describe("BaseItemsTable", () => {
       hasUnselected: true,
       onSelectAll,
       collection: createMockCollection({ can_write: true }),
+      onToggleSelected: jest.fn(),
```

Comment:
Changed the test here so things can pass - we should have the `onToggleSelected` function if we want to be able to select and deselect items. Refer to `canSelect` in the `DefaultItemsRenderer` component here (`frontend/src/metabase/components/ItemsTable/DefaultItemRenderer.tsx`)